### ISSUE TITLE
2025-01-29, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#66]
# Today's Arxiv Papers

(1) [AxBench: Steering LLMs? Even Simple Baselines Outperform Sparse Autoencoders](http://arxiv.org/abs/2501.17148v1), 2025-01-28

(2) [FactCG: Enhancing Fact Checkers with Graph-Based Multi-Hop Data](http://arxiv.org/abs/2501.17144v1), 2025-01-28

(3) [ASTRAL: Automated Safety Testing of Large Language Models](http://arxiv.org/abs/2501.17132v1), 2025-01-28

(4) [Histoires Morales: A French Dataset for Assessing Moral Alignment](http://arxiv.org/abs/2501.17117v1), 2025-01-28

(5) [Optimizing Large Language Model Training Using FP4 Quantization](http://arxiv.org/abs/2501.17116v1), 2025-01-28

(6) [Unlocking Transparent Alignment Through Enhanced Inverse Constitutional AI for Principle Extraction](http://arxiv.org/abs/2501.17112v1), 2025-01-28

(7) [Token-by-Token Regeneration and Domain Biases: A Benchmark of LLMs on Advanced Mathematical Problem-Solving](http://arxiv.org/abs/2501.17084v1), 2025-01-28

(8) [Enhanced Retrieval of Long Documents: Leveraging Fine-Grained Block Representations with Large Language Models](http://arxiv.org/abs/2501.17039v1), 2025-01-28

(9) [Challenges in Ensuring AI Safety in DeepSeek-R1 Models: The Shortcomings of Reinforcement Learning Strategies](http://arxiv.org/abs/2501.17030v1), 2025-01-28

(10) [Automated Refactoring of Non-Idiomatic Python Code: A Differentiated Replication with LLMs](http://arxiv.org/abs/2501.17024v1), 2025-01-28

(11) [Mobile Manipulation Instruction Generation from Multiple Images with Automatic Metric Enhancement](http://arxiv.org/abs/2501.17022v1), 2025-01-28

(12) [Large Language Models for Code Generation: The Practitioners Perspective](http://arxiv.org/abs/2501.16998v1), 2025-01-28

(13) [Artificial Intelligence Clones](http://arxiv.org/abs/2501.16996v1), 2025-01-28

(14) [Over-Tokenized Transformer: Vocabulary is Generally Worth Scaling](http://arxiv.org/abs/2501.16975v1), 2025-01-28

(15) [Instantiation-based Formalization of Logical Reasoning Tasks using Language Models and Logical Solvers](http://arxiv.org/abs/2501.16961v1), 2025-01-28

(16) [Multiple Abstraction Level Retrieve Augment Generation](http://arxiv.org/abs/2501.16952v1), 2025-01-28

(17) [ToolFactory: Automating Tool Generation by Leveraging LLM to Understand REST API Documentations](http://arxiv.org/abs/2501.16945v1), 2025-01-28

(18) [Detecting harassment and defamation in cyberbullying with emotion-adaptive training](http://arxiv.org/abs/2501.16925v1), 2025-01-28

(19) [RDMM: Fine-Tuned LLM Models for On-Device Robotic Decision Making with Enhanced Contextual Awareness in Specific Domains](http://arxiv.org/abs/2501.16899v1), 2025-01-28

(20) [Irony Detection, Reasoning and Understanding in Zero-shot Learning](http://arxiv.org/abs/2501.16884v1), 2025-01-28

(21) [JRE-L: Journalist, Reader, and Editor LLMs in the Loop for Science Journalism for the General Audience](http://arxiv.org/abs/2501.16865v1), 2025-01-28

(22) [Comparing Human and LLM Generated Code: The Jury is Still Out!](http://arxiv.org/abs/2501.16857v1), 2025-01-28

(23) [Adapting Network Information to Semantics for Generalizable and Plug-and-Play Multi-Scenario Network Diagnosis](http://arxiv.org/abs/2501.16842v1), 2025-01-28

(24) [Misspellings in Natural Language Processing: A survey](http://arxiv.org/abs/2501.16836v1), 2025-01-28

(25) [Exponential Family Attention](http://arxiv.org/abs/2501.16790v1), 2025-01-28

(26) [Exploring the Role of Explicit Temporal Modeling in Multimodal Large Language Models for Video Understanding](http://arxiv.org/abs/2501.16786v1), 2025-01-28

(27) [TORCHLIGHT: Shedding LIGHT on Real-World Attacks on Cloudless IoT Devices Concealed within the Tor Network](http://arxiv.org/abs/2501.16784v1), 2025-01-28

(28) [A Stochastic Dynamical Theory of LLM Self-Adversariality: Modeling Severity Drift as a Critical Process](http://arxiv.org/abs/2501.16783v1), 2025-01-28

(29) [HateBench: Benchmarking Hate Speech Detectors on LLM-Generated Content and Hate Campaigns](http://arxiv.org/abs/2501.16750v1), 2025-01-28

(30) [Through the Prism of Culture: Evaluating LLMs' Understanding of Indian Subcultures and Traditions](http://arxiv.org/abs/2501.16748v1), 2025-01-28

(31) [LLM Assisted Anomaly Detection Service for Site Reliability Engineers: Enhancing Cloud Infrastructure Resilience](http://arxiv.org/abs/2501.16744v1), 2025-01-28

(32) [Distilling Large Language Models for Network Active Queue Management](http://arxiv.org/abs/2501.16734v1), 2025-01-28

(33) [xJailbreak: Representation Space Guided Reinforcement Learning for Interpretable LLM Jailbreaking](http://arxiv.org/abs/2501.16727v1), 2025-01-28

(34) [3D-MoE: A Mixture-of-Experts Multi-modal LLM for 3D Vision and Pose Diffusion via Rectified Flow](http://arxiv.org/abs/2501.16698v1), 2025-01-28

(35) [Optimizing Code Runtime Performance through Context-Aware Retrieval-Augmented Generation](http://arxiv.org/abs/2501.16692v1), 2025-01-28

(36) [MACI: Multi-Agent Collaborative Intelligence for Robust Reasoning and Temporal Planning](http://arxiv.org/abs/2501.16689v1), 2025-01-28

(37) [MME-Industry: A Cross-Industry Multimodal Evaluation Benchmark](http://arxiv.org/abs/2501.16688v1), 2025-01-28

(38) [Auto-Differentiating Any LLM Workflow: A Farewell to Manual Prompting](http://arxiv.org/abs/2501.16673v1), 2025-01-28

(39) [VeriFact: Verifying Facts in LLM-Generated Clinical Text with Electronic Health Records](http://arxiv.org/abs/2501.16672v1), 2025-01-28

(40) [Jupybara: Operationalizing a Design Space for Actionable Data Analysis and Storytelling with LLMs](http://arxiv.org/abs/2501.16661v1), 2025-01-28

(41) [Contextual Reinforcement in Multimodal Token Compression for Large Language Models](http://arxiv.org/abs/2501.16658v1), 2025-01-28

(42) [Large Language Model Critics for Execution-Free Evaluation of Code Changes](http://arxiv.org/abs/2501.16655v1), 2025-01-28

(43) [DOCS: Quantifying Weight Similarity for Deeper Insights into Large Language Models](http://arxiv.org/abs/2501.16650v1), 2025-01-28

(44) [An LLM Benchmark for Addressee Recognition in Multi-modal Multi-party Dialogue](http://arxiv.org/abs/2501.16643v1), 2025-01-28

(45) [Why Do We Laugh? Annotation and Taxonomy Generation for Laughable Contexts in Spontaneous Text Conversation](http://arxiv.org/abs/2501.16635v1), 2025-01-28

(46) [CHiP: Cross-modal Hierarchical Direct Preference Optimization for Multimodal LLMs](http://arxiv.org/abs/2501.16629v1), 2025-01-28

(47) [Sparse Autoencoders Trained on the Same Data Learn Different Features](http://arxiv.org/abs/2501.16615v1), 2025-01-28

(48) [MCTS-SQL: An Effective Framework for Text-to-SQL with Monte Carlo Tree Search](http://arxiv.org/abs/2501.16607v1), 2025-01-28

(49) [Fine-Tuned Language Models as Space Systems Controllers](http://arxiv.org/abs/2501.16588v1), 2025-01-28

(50) [AffectGPT: A New Dataset, Model, and Benchmark for Emotion Understanding with Multimodal Large Language Models](http://arxiv.org/abs/2501.16566v1), 2025-01-27

(51) [Distributional Information Embedding: A Framework for Multi-bit Watermarking](http://arxiv.org/abs/2501.16558v1), 2025-01-27

(52) [Sample-Efficient Behavior Cloning Using General Domain Knowledge](http://arxiv.org/abs/2501.16546v1), 2025-01-27

(53) [Generalized Mission Planning for Heterogeneous Multi-Robot Teams via LLM-constructed Hierarchical Trees](http://arxiv.org/abs/2501.16539v1), 2025-01-27

(54) [Targeting Alignment: Extracting Safety Classifiers of Aligned LLMs](http://arxiv.org/abs/2501.16534v1), 2025-01-27

(55) [A comparison of data filtering techniques for English-Polish LLM-based machine translation in the biomedical domain](http://arxiv.org/abs/2501.16533v1), 2025-01-27

(56) [Programming by Examples Meets Historical Linguistics: A Large Language Model Based Approach to Sound Law Induction](http://arxiv.org/abs/2501.16524v1), 2025-01-27

(57) [How well can LLMs Grade Essays in Arabic?](http://arxiv.org/abs/2501.16516v1), 2025-01-27

(58) [Deception in LLMs: Self-Preservation and Autonomous Goals in Large Language Models](http://arxiv.org/abs/2501.16513v1), 2025-01-27

(59) [Smoothed Embeddings for Robust Language Models](http://arxiv.org/abs/2501.16497v1), 2025-01-27

(60) [Explaining GitHub Actions Failures with Large Language Models: Challenges, Insights, and Limitations](http://arxiv.org/abs/2501.16495v1), 2025-01-27

(61) [Generating customized prompts for Zero-Shot Rare Event Medical Image Classification using LLM](http://arxiv.org/abs/2501.16481v1), 2025-01-27

(62) [Cross-Domain Semantic Segmentation with Large Language Model-Assisted Descriptor Generation](http://arxiv.org/abs/2501.16467v1), 2025-01-27

(63) [On the Feasibility of Using LLMs to Execute Multistage Network Attacks](http://arxiv.org/abs/2501.16466v1), 2025-01-27

(64) [CoCoNUT: Structural Code Understanding does not fall out of a tree](http://arxiv.org/abs/2501.16456v1), 2025-01-27

(65) [Detecting Zero-Day Attacks in Digital Substations via In-Context Learning](http://arxiv.org/abs/2501.16453v1), 2025-01-27

(66) [Evaluating The Performance of Using Large Language Models to Automate Summarization of CT Simulation Orders in Radiation Oncology](http://arxiv.org/abs/2501.16309v1), 2025-01-27

(67) [RAPID: Retrieval-Augmented Parallel Inference Drafting for Text-Based Video Event Retrieval](http://arxiv.org/abs/2501.16303v1), 2025-01-27

(68) [Matryoshka Re-Ranker: A Flexible Re-Ranking Architecture With Configurable Depth and Width](http://arxiv.org/abs/2501.16302v1), 2025-01-27

(69) [Large Models in Dialogue for Active Perception and Anomaly Detection](http://arxiv.org/abs/2501.16300v1), 2025-01-27

(70) [FALCON: Resolving Visual Redundancy and Fragmentation in High-resolution Multimodal Large Language Models via Visual Registers](http://arxiv.org/abs/2501.16297v1), 2025-01-27

(71) [Brain-Adapter: Enhancing Neurological Disorder Analysis with Adapter-Tuning Multimodal Large Language Models](http://arxiv.org/abs/2501.16282v1), 2025-01-27

(72) [Do LLMs Have Visualization Literacy? An Evaluation on Modified Visualizations to Test Generalization in Data Interpretation](http://arxiv.org/abs/2501.16277v1), 2025-01-27

(73) [URAG: Implementing a Unified Hybrid RAG for Precise Answers in University Admission Chatbots -- A Case Study at HCMUT](http://arxiv.org/abs/2501.16276v1), 2025-01-27

(74) [Return of the Encoder: Maximizing Parameter Efficiency for SLMs](http://arxiv.org/abs/2501.16273v1), 2025-01-27

(75) [A foundation model for human-AI collaboration in medical literature mining](http://arxiv.org/abs/2501.16255v1), 2025-01-27

(76) [Multi-Agent Geospatial Copilots for Remote Sensing Workflows](http://arxiv.org/abs/2501.16254v1), 2025-01-27

(77) [Zero-Shot Decision Tree Construction via Large Language Models](http://arxiv.org/abs/2501.16247v1), 2025-01-27

(78) [Phase Transitions in Large Language Models and the $O(N)$ Model](http://arxiv.org/abs/2501.16241v1), 2025-01-27

(79) [AiGet: Transforming Everyday Moments into Hidden Knowledge Discovery with AI Assistance on Smart Glasses](http://dx.doi.org/10.1145/3706598.3713953), 2025-01-27

(80) [Language-Based Bayesian Optimization Research Assistant (BORA)](http://arxiv.org/abs/2501.16224v1), 2025-01-27

(81) [DBRouting: Routing End User Queries to Databases for Answerability](http://arxiv.org/abs/2501.16220v2), 2025-01-27

(82) [Enhancing Visual Inspection Capability of Multi-Modal Large Language Models on Medical Time Series with Supportive Conformalized and Interpretable Small Specialized Models](http://arxiv.org/abs/2501.16215v1), 2025-01-27

(83) [Provence: efficient and robust context pruning for retrieval-augmented generation](http://arxiv.org/abs/2501.16214v1), 2025-01-27

(84) [From Informal to Formal -- Incorporating and Evaluating LLMs on Natural Language Requirements to Verifiable Formal Proofs](http://arxiv.org/abs/2501.16207v1), 2025-01-27

(85) [Raiders of the Lost Dependency: Fixing Dependency Conflicts in Python using LLMs](http://arxiv.org/abs/2501.16191v1), 2025-01-27

(86) [SWIFT: Mapping Sub-series with Wavelet Decomposition Improves Time Series Forecasting](http://arxiv.org/abs/2501.16178v1), 2025-01-27

(87) [Will Systems of LLM Agents Cooperate: An Investigation into a Social Dilemma](http://arxiv.org/abs/2501.16173v1), 2025-01-27

(88) [MetaDecorator: Generating Immersive Virtual Tours through Multimodality](http://arxiv.org/abs/2501.16164v1), 2025-01-27

(89) [CITYWALK: Enhancing LLM-Based C++ Unit Test Generation via Project-Dependency Awareness and Language-Specific Knowledge](http://arxiv.org/abs/2501.16155v1), 2025-01-27

(90) [AdaCoT: Rethinking Cross-Lingual Factual Reasoning through Adaptive Chain-of-Thought](http://arxiv.org/abs/2501.16154v1), 2025-01-27

(91) [AI Agents for Computer Use: A Review of Instruction-based Computer Control, GUI Automation, and Operator Assistants](http://arxiv.org/abs/2501.16150v1), 2025-01-27

(92) [PATCH: Empowering Large Language Model with Programmer-Intent Guidance and Collaborative-Behavior Simulation for Automatic Bug Fixing](http://arxiv.org/abs/2501.16149v1), 2025-01-27

(93) [SampleLLM: Optimizing Tabular Data Synthesis in Recommendations](http://arxiv.org/abs/2501.16125v1), 2025-01-27

(94) [The Shiny Scary Future of Automated Research Synthesis in HCI](http://arxiv.org/abs/2501.16084v1), 2025-01-27

(95) [Integration of LLM Quality Assurance into an NLG System](http://arxiv.org/abs/2501.16078v1), 2025-01-27

(96) [PISCO: Pretty Simple Compression for Retrieval-Augmented Generation](http://arxiv.org/abs/2501.16075v1), 2025-01-27

(97) [Skeleton-Guided-Translation: A Benchmarking Framework for Code Repository Translation with Fine-Grained Quality Evaluation](http://arxiv.org/abs/2501.16050v1), 2025-01-27

(98) [PRISMe: A Novel LLM-Powered Tool for Interactive Privacy Policy Assessment](http://arxiv.org/abs/2501.16033v1), 2025-01-27

(99) [FDLLM: A Text Fingerprint Detection Method for LLMs in Multi-Language, Multi-Domain Black-Box Environments](http://arxiv.org/abs/2501.16029v1), 2025-01-27

(100) [TOPLOC: A Locality Sensitive Hashing Scheme for Trustless Verifiable Inference](http://arxiv.org/abs/2501.16007v1), 2025-01-27
